build.py

```diff
@@ -1,25 +1,20 @@
+import os
 import sys
+import subprocess
 from pathlib import Path
 from contextlib import chdir
-import subprocess
-import os
 
-
-def usage() -> None:
-    """Print usage information for the script."""
-
-    print("Usage: python build.py [OPTIONS]", end="\n\n")
+def usage():
+    print("Usage: build.py [options]")
     print("Options:")
-    print("  -h | --help                 Show this message")
-    print("  -j                          Number of processors to use [Default: 1]")
-    print("  -c | --clean                Clean after build")
-    print("  --build-dir                 Build directory [Default: build]")
-    print("  --no-tests                  Don't build tests")
-    print("  --cxx-std                   C++ standard for compilation [Default: 14]")
-    print(
-        "  --build-type                Release, Debug, RelWithDebInfo [Default: Release]"
-    )
-
+    print("  -h, --help            Show this help message and exit")
+    print("  -j N                  Number of processors to use")
+    print("  -c, --clean           Clean build")
+    print("  --build-dir DIR       Build directory")
+    print("  --no-tests            Do not build tests")
+    print("  --cxx-std STD         C++ standard to use")
+    print("  --build-type TYPE     Build type (e.g., Release, Debug)")
+    print("  --output-to-file      Output logs to file instead of terminal")
```

Comment:
In this block, why did you remove the test on default output?

Reply:
Sorry I don't get, what do you mean "test on default output"? If you mean specifying the default in the output message just forgot to include them

Reply:
It seems that you modified the text compared to the original version. For instance:

Number of processors to use [Default: 1]

was changed to:

Number of processors to use

which provides less information

Reply:
Fixed in commit